src/utilities/util_verify.c

```diff
@@ -61,14 +61,16 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
     size_t size;
     int ch;
     char *config, *dump_offsets, *key, *uri;
-    bool do_not_clear_txn_id, dump_address, dump_app_data, dump_blocks, dump_layout, dump_pages,
-      read_corrupt, stable_timestamp;
+    bool abort_on_error, do_not_clear_txn_id, dump_address, dump_app_data, dump_blocks, dump_layout,
+      dump_pages, read_corrupt, stable_timestamp;
 
-    do_not_clear_txn_id = dump_address = dump_app_data = dump_blocks = dump_layout = dump_pages =
-      read_corrupt = stable_timestamp = false;
+    abort_on_error = do_not_clear_txn_id = dump_address = dump_app_data = dump_blocks =
+      dump_layout = dump_pages = read_corrupt = stable_timestamp = false;
     config = dump_offsets = uri = NULL;
-    while ((ch = __wt_getopt(progname, argc, argv, "cd:stu?")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "acd:stu?")) != EOF)
         switch (ch) {
+        case 'a':
+            abort_on_error = true;
```

Comment:
Missing `break;`

Reply:
Added a break statement in the second commit.

Reply:
Ugh! I could have sworn I was looking at the "all commits" view. But apparently not...